media-libs/game-music-emu/files/gme-drop-cmake-cxxflags.patch

```diff
@@ -0,0 +1,31 @@
+From 1096cfa2e6bb00ff46b76b870be8d1ece9154815 Mon Sep 17 00:00:00 2001
+From: Vivian Heisz (vhz) <demize@unstable.systems>
+Date: Thu, 22 May 2025 20:21:57 -0400
+Subject: [PATCH] Drop -fno-rtti and -fno-exceptions forced CXXFLAGS
+
+Signed-off-by: Vivian Heisz (vhz) <demize@unstable.systems>
+---
+ CMakeLists.txt | 7 -------
+ 1 file changed, 7 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 9128a6e..1cd8acc 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -87,13 +87,6 @@ if(CMAKE_BUILD_TYPE_LOWER STREQUAL "debug")
+     endif()
+ endif()
+ 
+-# c++ tweaks
+-if(CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
+-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti -fno-exceptions")
```

Comment:
It's okay to keep these if a project knows it doesn't use them. I don't think we need to be removing them, unless there's some particular reason?

Reply:
I was trying to avoid QA0069, and tried to check how other packages handle it; I could see some packages going both ways, but I erred on the side of removing them.

I'll defer to you on it though, so I've removed this patch now.

Reply:
Probably the key distinction here is that `-fno-rtti -fno-exceptions` aren't really flags users should or would ever set globally and it depends on how the codebase uses (or doesn't use) those features. If upstream build w/ them disabled, there's no point in ever allowing them to be enabled in this instance.